lib/ddtrace/contrib/active_record/utils.rb

```diff
@@ -0,0 +1,85 @@
+module Datadog
+  module Contrib
+    module ActiveRecord
+      # Common utilities for Rails
+      module Utils
+        # Return a canonical name for a type of database
+        def self.normalize_vendor(vendor)
+          case vendor
+          when nil
+            'defaultdb'
+          when 'mysql2'
+            'mysql'
```

Comment:
I would prefer to not include this change otherwise users with default settings will get their services renamed. Let's keep the current value that is `mysql2`.